credentials/ssocreds/os_windows.go

```diff
@@ -0,0 +1,7 @@
+package ssocreds
+
+import "os"
+
+func getHomeDirectory() string {
+	return os.Getenv("USERPROFILE")
+}
```

Comment:
Assuming this does not require  ```// +build windows``` at the top, as other file has  ```// +build !windows```

Reply:
I think the `windows` part is implicit because of the filename. it doesn't hurt to have `// +build windows`, but i think the Go compiler is supossed to figure it out in these cases. Though linux tests shoudl fail if this was not the case.

Reply:
Correct the _GOOS at the tail end of the file before the extension is an implicit build flag.